agent/src/beerocks/slave/backhaul_manager/wan_monitor.cpp

```diff
@@ -33,7 +33,7 @@ static int netlink_open_socket()
     // Open a netlink socket
     int fd = socket(AF_NETLINK, SOCK_RAW, NETLINK_ROUTE);
     if (fd < 0) {
-        LOG(ERROR) << "Failed creating netlink socket";
+        LOG(ERROR) << "Failed creating netlink socket: " << strerror(errno);
```

Comment:
Good - will this print that there are no permissions to create the socket?

Reply:
yes